rpm/topology.spec

```
@@ -0,0 +1,31 @@
+Summary: osg-notify tool
+Name: osg-notify
```

```suggestion
Name: topology-client
```

As requested in https://opensciencegrid.atlassian.net/browse/SOFTWARE-3854